.github/workflows/vuln-scan.yaml

```diff
@@ -182,9 +182,10 @@ jobs:
       # ref: https://github.com/peter-evans/create-pull-request
       - name: Create or update a PR
         if: steps.analyze.outputs.proceed == 'yes'
-        uses: peter-evans/create-pull-request@v3
+        uses: peter-evans/create-pull-request@052fc72b4198ba9fbc81b818c6e1859f747d49a8
         with:
           token: "${{ secrets.GITHUB_TOKEN }}"
+          author: jupyterhub vuln-scan bot <noreply@github.com>
```

Comment:
I think this LGTM, I wonder what the outcome of not setting the committer and leaving it as the default of `GitHub <noreply@github.com>`.